lms/envs/aws.py

```diff
@@ -558,3 +558,8 @@
 ##### CDN EXPERIMENT/MONITORING FLAGS #####
 PERFORMANCE_GRAPHITE_URL = ENV_TOKENS.get('PERFORMANCE_GRAPHITE_URL', PERFORMANCE_GRAPHITE_URL)
 CDN_VIDEO_URLS = ENV_TOKENS.get('CDN_VIDEO_URLS', CDN_VIDEO_URLS)
+
+##### ECOMMERCE API CONFIGURATION SETTINGS #####
+ECOMMERCE_API_URL = ENV_TOKENS.get("ECOMMERCE_API_URL")
+ECOMMERCE_API_SIGNING_KEY = AUTH_TOKENS.get("ECOMMERCE_API_SIGNING_KEY")
+ECOMMERCE_API_TIMEOUT = ENV_TOKENS.get("ECOMMERCE_API_TIMEOUT", 5)
```

Comment:
Looks good, my only nitpick is to use single quotes instead of double quotes for these strings.